src/app/app.component.scss

```diff
@@ -13,6 +13,18 @@
 	padding: $padding * 0.25 $padding * 0.25 $padding * 0.25 $padding * 0.5;
 }
 
+.date-wrapper {
+	display: flex;
+	align-items: center;
+	justify-content: space-evenly;
+	flex: 1;
+
+	span {
```

Comment:
margin-bottom: 6px seems a bit weird, is the better solved with line-height?

Reply:
No. Removing `margin-bottom: 6px ` will make the `span` element positioning a bit below as the "Downloads in past" text for the mat-select starts

Reply:
bump

Reply:
bump?

Reply:
removed in [b869c6e](https://github.com/bitovi/opensource-analytics-dashboard/pull/15/commits/b869c6efa2f30cadc3e337e46be9ab1ea14ed76f)

Reply:
can you add more context? What was it for? lol temp code?

Reply:
> In the end I removed margin-bottom: 6px in this change [b869c6e](https://github.com/bitovi/opensource-analytics-dashboard/pull/15/commits/b869c6efa2f30cadc3e337e46be9ab1ea14ed76f) from the dropdown.scss and app.scss and the result was that everything is aligned in center

kk that makes sense, resolved